travel-agency-persistence/src/main/java/cz/fi/muni/pa165/travelagency/entity/Excursion.java

```diff
@@ -29,5 +60,114 @@ public Long getId() {
     public void setId(Long id) {
         this.id = id;
     }
+
+    public Trip getTrip() {
+                return trip;
+    }
+
+    public void setTrip(Trip trip) {
+                this.trip = trip;
+    }
     
+    public Date getDate() {
+        return date;
+    }
+
+    public void setDate(Date date) {
+        this.date = date;
+    }
+
+    public Duration getDuration() {
+        return duration;
+    }
+
+    public void setDuration(Duration duration) {
+        this.duration = duration;
+    }
+    
+    public String getDescription() {
+        return description;
 }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public String getDestination() {
+        return destination;
+    }	
+
+    public void setDestination(String destination) {
+        this.destination = destination;
+    }
+    
+    public BigDecimal getPrice() {
+        return price;
+    }
+
+    public void setPrice(BigDecimal price) {
+        this.price = price;
+    }
+    
+    @Override
+    public int hashCode() {
+            final int prime = 31;
+            int result = 1;
+            result = prime * result + ((date == null) ? 0 : date.hashCode());
+            result = prime * result + ((description == null) ? 0 : description.hashCode());
+            result = prime * result + ((destination == null) ? 0 : destination.hashCode());
+            result = prime * result + ((duration == null) ? 0 : duration.hashCode());
+            result = prime * result + ((price == null) ? 0 : price.hashCode());
+            result = prime * result + ((trip == null) ? 0 : trip.hashCode());
+            return result;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+            if (this == obj)
+                    return true;
+            if (obj == null)
+                    return false;
+            if (getClass() != obj.getClass())
+                    return false;
+            Excursion other = (Excursion) obj;
+            if (date == null) {
+                    if (other.date != null)
+                            return false;
+            } else if (!date.equals(other.date))
+                    return false;
```

Comment:
Nepristupuj k atributom other priamo, moze to byt proxy trieda. Pouzi vsade gettery.